qcs/src/configuration/settings.rs

```diff
@@ -1,21 +1,23 @@
 use std::collections::HashMap;
 
-use eyre::{Result, WrapErr};
 use serde::{Deserialize, Serialize};
 
 use super::path::path_from_env_or_home;
+use super::LoadError;
 
 /// Setting this environment variable will change which file is used for loading settings
 pub const SETTINGS_PATH_VAR: &str = "QCS_SETTINGS_FILE_PATH";
```

Comment:
nit: in the docstring, it might be helpful to repeat the value for this.. because of the const, both are in caps & I had to think about it for more than 5 seconds: "Is it `$SETTINGS_PATH_VAR`, or `$QCS_SETTINGS_FILE_PATH`..."
```diff
- /// Setting this environment variable will change which file is used for loading settings
+ /// Setting the `QCS_SETTINGS_FILE_PATH` environment variable will change which file is used for loading settings
```

(Same for the other env vars defined by consts)

Reply:
Yeah... I originally resisted repeating so that it wouldn't fall out of sync if we change the value but it's _very_ unlikely we'll change this (it's kept in line with pyquil today).